.ci/docker/buildtools-windows/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# escape=`
+
+# Use the latest Windows Server Core image with .NET Framework 4.8.
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-1909
```

Comment:
For the time being this particular implementation is only for beta purposes, it's not yet ready but the framework is in place so we are working on supporting it but it might happen after this particular PR is merged.